### DISCUSSION TITLE
Issues with Update 20210403

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#290] [Fixed Issues]
Updated my 64bit installation, rebooted, and now have no wifi connection. The netctl/SSID file looks unchanged. I have not rebooted after previous updates, so it could be due to an earlier change.

**Comment by R101G** [comment]
Creating a wifi file in /boot and powering up has restored connection, so it looks as if the update altered a wifi setting. I am using a USB dongle, which might make a difference.

**Reply by R101G** [reply]
Seems the problem is that rAudio will not now reconnect without a wifi file present in /boot.

**Comment by R101G** [comment]
A new install with the latest image seems to get rid of the problem. There must an easier way, though.

**Comment by bnilsson11** [comment]
Since 20210403 I can no longer setup a NAS mount.
Clicking on the NAS symbol just results in a forever wait flash.
If there is another method to do it, I haven't found it.

**Reply by bnilsson11** [reply]
How can I set up an NFS mount by command line?

**Reply by rern** [reply]
`20210414` update should fix mount/remount/unmount network share bug.
